README.md

```diff
@@ -148,6 +148,7 @@ The following optional inputs:
 | --- | --- |
 | `cosign-release` | `cosign` version to use instead of the default. |
 | `install-dir` | directory to place the `cosign` binary into instead of the default (`$HOME/.cosign`). |
+| `use-sudo` | set to `true` if `install-dir` location requires sudo privs. |
```

Comment:
can you add the comment that the default is set to false? thanks!

Reply:
Good catch, this is updated now